practice_projects/#Homework_pt.py

```diff
@@ -0,0 +1,34 @@
+#Homework pt.1
+
+Friend= "\nSoftware is my friend!\n"
+print (Friend)
+
+#Homework pt.2
+
+Soft = "Software is"
+my = "my friend\n"
+print (Soft,my)
+
+#Homework pt.3
+
+variable_1 = "I would like this to be a string\n"
```

Comment:
Good. You don't need a newline after, but that's your preference so totally fine.